### ISSUE TITLE
React Transition Group 4.4.3 Error

### ISSUE THREAD

**Issue by Jak2k** [primefaces/primereact#3120]
### Describe the bug

When I build my app I'll get
```
Module not found: Error: Can't resolve 'react-transition-group' in 'node_modules\primereact\csstransition'
```

### PrimeReact version

8.2.0

### React version

18.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Steps to reproduce the behavior

1 `npm i`
2. `npm run build`

### Expected behavior

It should just build.

**Comment by melloware** [comment]
ITs a bug in `react-transition-group` 4.4.3 please back up to 4.4.2 see: https://github.com/reactjs/react-transition-group/issues/844

**Comment by Jak2k** [comment]
How can I do that? I think that should be solved in primereacts's package.json.

(When I use pnpm, everything works.)

**Comment by melloware** [comment]
Can you post your package.json here?

**Comment by melloware** [comment]
PrimeReact has this in its package.json
```json
"react-transition-group": "^4.4.1",
```
Which means any version newer than 4.4.1 and yours is bringing in 4.4.3 which is the bad version just deployed to by that team.

To fix it simply add this in your package.json....

```json
"react-transition-group": "4.4.2",
```

This locks it to 4.4.2 and tells it not to get the higher version and overrides what is in PrimeReact's JSON.

**Comment by melloware** [comment]
Actually looks like that teams just published 4.4.4 which fixes the issue.

**Comment by mertsincan** [comment]
Bad luck ;) They released a new version 3 hours after v4.4.3 was released. @melloware Maybe, we can add a fixed version for such problems. WDYT?

**Comment by melloware** [comment]
They marked 4.4.3 deprecated.  I joked this is the definition of a "supply chain attack".  Not sure what we can do to remedy this in the future if someone releases a bad version?
